### DISCUSSION TITLE
Sound improvements à la Soundproof Walls

### DISCUSSION THREAD

**Discussion by rekterakathom** [FakeFishGames/Barotrauma#15442] [Ideas and suggestions]
## The soundscape problem
The current soundscape of the game is cluttered and non-immersive. It does not feel like we are on a submarine with thick metal walls everywhere when I can hear the reactor or my friends talking no matter where I am on the submarine.

This video (and the mod it is demonstrating, Soundproof Walls) demonstrate what I mean:
https://www.youtube.com/watch?v=UOSuh0ncEn0

So I propose that the vanilla game takes inspiration from this mod to improve the soundscape of the game.

## The VOIP problem
The sound quality feels pretty bad when using the in-game voice system. I don't know the bit-rate / codec, but it *feels* bad. Whether this is actually due to a very low bit-rate or just bad effects / filtering, I don't know.

To the above I also want to add that the current radio "effect" is horrible. It does not sound like a radio, it sounds like the bit-rate was just cut in half.

In addition to the perceived bad quality, voice transmission is very stuttery. Players with even moderately high ping (100+) are almost undecipherable because their speech transmission stutters so much.

Edit: I also wanted to mention the directionality of speech over the radio. In my opinion, radio should not convey direction at all. Maybe if the speaker is within speech distance, and even then the panning should be toned down. They don't do it in real life, they shouldn't do it here. In addition to realism, this would give more of an incentive to actually use local voice instead of the radio. Currently the only reason I use local at all is if I am doing traitor stuff or I have a migraine from the horrible quality of the radio.

**Comment by NotWendy** [comment]
Thank you for mentioning the radio quality. That's the reason I myself don't use in-game voice chat. It intentionally crushes the audio quality, which has catastrophic results for players with high ping or a bad microphone.
